docs/config.json

```diff
@@ -830,6 +830,10 @@
           "title": "Introduction",
           "slug": "/desktop-access/introduction/"
         },
+        {
+          "title": "Desktop Discovery",
```

Comment:
I don't want to confuse this with the automatic LDAP-based discovery of desktops.

IMHO, this new guide should be the default a user sees under "Getting Started." The original getting started guide should be moved to a page called "manual setup" or something.